playwright/components/link-preview/index.ts

```diff
@@ -0,0 +1,30 @@
+import type { Page } from "@playwright/test";
+import { PILL_CLOSE_ICON } from "../locators";
+import { dlsRoot } from "../../../playwright/components/index";
+import { LinkPreviewProps } from "../../../src/components/link-preview";
+
+// component preview locators
+export const linkPreview = (page: Page) => {
+  return dlsRoot(page).locator("a");
```

Comment:
I don't think you need to specify `dlsRoot()` here as `locator("a")` is in there and it should find it. This was a comment James made on my Note PR which uses LinkPreview :)